### PR TITLE
Execute "capture view" at the last

### PR DESCRIPTION
**Pull request by MingboPeng**
Bring it to the front.

### CODE REVIEW

**Comment by mostaphaRoudsari** [comment]
Hi @MingboPeng, This is great. I only have one concern and I'm not sure what is the best way to address it.

With the current set up, the component will update Grasshopper's order of execution on each run. It can be a problem if user is running the component in a huge file for iterative runs. Can we check if the component is already on top and then if it's not reorder the components?

**Comment by MingboPeng** [comment]
@mostaphaRoudsari , thanks for your point. Yes, we should check this. I will submit a new pull request.

**Comment by MingboPeng** [comment]
@mostaphaRoudsari updated.

File: src/Ladybug_Capture View.py [hunk 1]
```diff
@@ -130,14 +132,24 @@ def viewCapture(fileName, directory, viewNames, image_width, image_height, dispM
...
+        
 
 if _capture and _fileName!=None:
     
     directory = mdPath(folder_)
     
+    doc = GH.Instances.ActiveCanvas.Document.Objects;
+    isAtLast = doc[doc.Count-1].InstanceGuid.Equals(ghenv.Component.InstanceGuid);
+    if (not isAtLast):
+        # bring this to the top
```
Comment (mostaphaRoudsari) [inline src/Ladybug_Capture View.py]: Hi @MingboPeng! Thanks. Looks great! There are two/three styling issues which should be very easy to fix.
1. No ; at the end of line 141 and 142.
2. No ( ) after if in line 143.
I blame both on C#  and javascript ;)

Comment (MingboPeng) [inline src/Ladybug_Capture View.py]: @mostaphaRoudsari , it is fixed. Haha, python's styling, styling.

**Comment by mostaphaRoudsari** [comment]
Thanks @MingboPeng! Merged! :)

**Comment by chriswmackey** [comment]
Thanks @MingboPeng !  This is a great idea.

**Pull request merged**